src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2002.ProvideMessageArguments_Invalid.sarif

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SARIF Functional Testing",
+          "version": "1.2.3"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "TEST1001",
+          "message": {
+            "text": "Test 1001 full description.",
+            "arguments": [
```

Comment:
>arguments [](start = 13, length = 9)

Remove this. Your test logic is correct: this _is_ a violation because `id` is missing. But `arguments` is  unnecessary for the test, and its presence might confuse someone reading this test file. It confused me :-)